mypy/nodes.py

```diff
@@ -3017,6 +3017,24 @@ class is generic then it will be a type constructor of higher kind.
         "is_intersection",
     ]
 
+    # Special attributes not collected as protocol members by Python 3.12
+    # See typing._SPECIAL_NAMES
+    EXCLUDED_ATTRIBUTES: Final = frozenset(
```

Comment:
I think that module level constant is better :)

Reply:
I moved it back above the class. Unless it should go somewhere else.